app/Http/Controllers/RequestWithdrawalController.php

```
@@ -0,0 +1,156 @@
+            }
+
+            return RequestWithdrawalListingResource::collection($withdrawals)
+                ->additional([
+                    'message' => 'Request Withdrawals retrieved successfully.',
+                    'success' => true,
+                ]);
+        } catch (Throwable $e) {
+            Log::error('Error fetching withdrawals', [
+                'error' => $e->getMessage(),
+                'trace' => $e->getTraceAsString(),
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'index'. (Clean Code Rules)

(StaticAccess)